DataTable/Table/Layout/CustomLayout.php

```diff
@@ -2,23 +2,25 @@
 
 namespace CrossKnowledge\DataTableBundle\DataTable\Table\Layout;
 
-
 class CustomLayout implements DataTableLayoutInterface
 {
-    protected $name, $dom;
+    protected string $dom;
```

Comment:
again check that this could never be null, use ?string if you are not sure

Reply:
Corrected. Done.